js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -617,7 +617,23 @@ const DropDownEditor = TextBox.inherit({
     },
 
     _popupPositionedHandler: function(e) {
-        e.position && this._popup.$overlayContent().toggleClass(DROP_DOWN_EDITOR_OVERLAY_FLIPPED, e.position.v.flip);
+        const { labelMode, stylingMode } = this.option();
+
+        const $popupOverlayContent = this._popup.$overlayContent();
+        const isOverlayFlipped = e.position.v.flip;
+        const shouldIndentForLabel = labelMode !== 'hidden' && stylingMode === 'outlined';
+
+        if(e.position) {
+            $popupOverlayContent.toggleClass(DROP_DOWN_EDITOR_OVERLAY_FLIPPED, isOverlayFlipped);
+        }
+
+        if(isOverlayFlipped && shouldIndentForLabel && this._label.isVisible()) {
```

Comment:
Hm, i think `this._label.isVisible()` is always `true` if `labelMode !== 'hidden'`, not?

Reply:
[commit](https://github.com/DevExpress/DevExtreme/pull/22471/commits/eb90394b50ac98160b9f9b8a15c21453b70813d6)